openstack/networking/v2/extensions/external/results.go

```diff
@@ -1,76 +1,8 @@
 package external
 
-import (
-	"github.com/gophercloud/gophercloud/openstack/networking/v2/networks"
-	"github.com/gophercloud/gophercloud/pagination"
-)
-
-// NetworkExternal represents a decorated form of a Network with based on the
+// NetworkExt represents a decorated form of a Network with based on the
 // "external-net" extension.
-type NetworkExternal struct {
-	// UUID for the network
-	ID string `json:"id"`
-
-	// Human-readable name for the network. Might not be unique.
-	Name string `json:"name"`
-
-	// The administrative state of network. If false (down), the network does not forward packets.
-	AdminStateUp bool `json:"admin_state_up"`
-
-	// Indicates whether network is currently operational. Possible values include
-	// `ACTIVE', `DOWN', `BUILD', or `ERROR'. Plug-ins might define additional values.
-	Status string `json:"status"`
-
-	// Subnets associated with this network.
-	Subnets []string `json:"subnets"`
-
-	// Owner of network. Only admin users can specify a tenant_id other than its own.
-	TenantID string `json:"tenant_id"`
-
-	// Specifies whether the network resource can be accessed by any tenant or not.
-	Shared bool `json:"shared"`
-
+type NetworkExt struct {
```

Comment:
I think extension names should be renamed to something more unique. Out of curiosity, I tried to mix two extensions in like so:

```go
type NetworkWithExt {
        networks.Network
        provider.NetworkExt
        external.NetworkExt
}
```

And got a Go error of:

```
duplicate field NetworkExt
```